<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseConversion</class>
 <widget class="QWidget" name="BaseConversion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_B">
     <property name="text">
      <string>二进制：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_B"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_O">
     <property name="text">
      <string>八进制：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_O"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_D">
     <property name="text">
      <string>十进制：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_D"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_H">
     <property name="text">
      <string>十六进制：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_H"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
